client/components/data/query-reader-start-recommendations/index.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingRecommendations, getRecommendationsInteractedWith } from 'state/reader/start/selectors';
+import { requestRecommendations } from 'state/reader/start/actions';
+
+class QueryReaderStartRecommendations extends Component {
+	componentWillMount() {
```

Comment:
I've left this out for now - will add in a later PR when we add the action for recommendations interacted with.